### PR TITLE
Add the Sub Sections Under the "Using the Plugin" Section

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to add the sub sections under the "Using the Plugin" section in [1].

[1] https://ballerina.io/learn/tools-ides/intellij-plugin/using-the-intellij-plugin
> Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/141

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/tools-ides/intellij-plugin.md [hunk 1]
```diff
@@ -95,7 +95,12 @@ After obtaining the ZIP file using either of the above approaches, follow the st
 
 ## Using the plugin
 
-For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](/learn/tools-ides/intellij-plugin/using-the-intellij-plugin).
+Click on the below links to find information on using the IntelliJ Ballerina plugin to write Ballerina programs.
```
Comment (anupama-pathirage) [inline learn/tools-ides/intellij-plugin.md]: Do we need to say "Click on the .."? IMO, something like "Refer below links" is more appopriate.

Comment (praneesha) [reply learn/tools-ides/intellij-plugin.md]: Normally, we don't say "refer" when the user has to click on the links. Therefore, changed as follows:

- The below sections include information on using the IntelliJ Ballerina plugin to write Ballerina programs.
- The below sections include information on the various capabilities that are facilitated by the IntelliJ Ballerina plugin for the development process.

Comment (anupama-pathirage) [reply learn/tools-ides/intellij-plugin.md]: We need to change the same in the relevant pages as well.
https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md

Comment (praneesha) [reply learn/tools-ides/intellij-plugin.md]: Changed in the below files:

- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin.md
- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/vscode-plugin.md
 - https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/vscode-plugin/graphical-editor.md
 - https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/vscode-plugin/run-all-tests.md

File: learn/tools-ides/intellij-plugin.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ permalink: /learn/tools-ides/intellij-plugin/
 
 # The IntelliJ IDEA Ballerina Plugin
 
-The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. Click on the below links for instructions on how to download, install, and use the features of the IntelliJ plugin.
+The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. The below links sections include instructions on how to download, install, and use the features of the IntelliJ plugin.
```
Comment (anupama-pathirage) [inline learn/tools-ides/intellij-plugin.md]:
```suggestion
The IntelliJ Ballerina plugin provides the Ballerina development capabilities in IntelliJ IDEA. The below sections include instructions on how to download, install, and use the features of the IntelliJ plugin.
```

**Pull request merged**